src/components/Dialog.js

```diff
@@ -121,11 +127,11 @@ export function Dialog({
         </h2>
         {onCancel && (
           <div className="Hyp-Dialog__close">
-            <IconButton icon="cancel" title="Close" onClick={onCancel} />
+            <IconButton icon="hyp-cancel" title="Close" onClick={onCancel} />
           </div>
         )}
       </header>
-      <div>{children}</div>
+      {children}
```

Comment:
This wrapping element needed to be removed such that `{children}`'s elements were immediate children of the `Dialog`, for overflow reasons. I will double-check this change against the `client` to assure it hasn't caused any layout changes there.